src/main/java/org/commcare/cases/query/QueryCacheEntry.java

```diff
@@ -0,0 +1,13 @@
+package org.commcare.cases.query;
+
+/**
+ * A query cache stores data local to a query context which will be relevant for recomputation
+ * within the lifecycle of the Query.
+ *
+ * Note that there can only be one instance of a query cache at any given time
+ *
+ * Created by ctsims on 1/26/2017.
+ */
+
+public interface QueryCacheEntry {
```

Comment:
Updated the docs. This is a tag interface used to specify the lifecycle of the object